pkgs/tools/networking/openssh/common.nix

```diff
@@ -37,6 +37,8 @@ stdenv.mkDerivation rec {
   patches = [
     ./locale_archive.patch
 
+    ./fix-undeclared-maxhostnamelen.patch
```

Comment:
Can we fetch this from alpine (https://git.alpinelinux.org/aports/plain/main/openssh/gss-serv.c.patch?id=a7509603971ce2f3282486a43bb773b1b522af83) instead of making our own copy?

(Since it's a static file in Alpine's repo, we can use `fetchurl` instead of `fetchpatch`, so it doesn't matter if openssh is in the bootstrap chain of `fetchpatch` — I think it might be.)